src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -99,6 +99,11 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
             {
                 return;
             }
+            if (UseManagedHandler)
```

Comment:
nit: newline after '}'

Reply:
done. this is how it was before. I did simple revert and replace issue number.